src/util/text.js

```diff
@@ -24,7 +25,8 @@ function estimate(text) {
 
 // measure text width if canvas is available
 function measureWidth(item) {
-  return context.font = font(item), measure(textValue(item));
+  font(item);
+  return context.font = measure(textValue(item));
```

Comment:
oops, must have been getting tired with some of the rewrites.  Thanks, fixing...